quickjs.c

```diff
@@ -36078,6 +36172,31 @@ static const JSCFunctionListEntry js_error_proto_funcs[] = {
     JS_PROP_STRING_DEF("message", "", JS_PROP_WRITABLE | JS_PROP_CONFIGURABLE ),
 };
 
+static JSValue js_error_get_prepareStackTrace(JSContext *ctx, JSValue this_val)
+{
+    JSValue val, ret;
+
+    val = JS_ToObject(ctx, this_val);
+    if (JS_IsException(val))
+        return val;
+    ret = JS_GetPrototype(ctx, val);
```

Comment:
I confess I don't understand how this returns `ctx->error_prepare_stack` :)

Reply:
Well, that's werid lol. I can simplify that of course.